orderer/common/multichannel/registrar.go

```diff
@@ -1015,23 +1045,15 @@ func (r *Registrar) removeSystemChannel() error {
 		}
 		if !isChannelMember {
 			logger.Debugf("Not a member of channel %s, removing it", channel)
-			err := r.removeMember(channel, cs)
+			err := r.ledgerFactory.Remove(channel)
 			if err != nil {
-				return errors.WithMessagef(err, "failed to remove channel: %s", channel)
+				return errors.WithMessagef(err, "failed removing ledger for channel %s", channel)
```

Comment:
Can we try all of them and not stop on the first? We can return an error at the end if one or more failed, but how about trying to remove them all? this would at least stop all the ones that needs stopping, and won't keep the server with channels running when they are still expecting the system channel to be there.

Reply:
Yes now it attempts all the removals before erring